go/vt/mysqlctl/mysql_daemon.go

```diff
@@ -284,11 +287,6 @@ func (fmd *FakeMysqlDaemon) GetSchema(dbName string, tables, excludeTables []str
 // GetDbConnection is part of the MysqlDaemon interface
 func (fmd *FakeMysqlDaemon) GetDbConnection(dbconfigName dbconfigs.DbConfigName) (dbconnpool.PoolConnection, error) {
```

Comment:
This probably doesn't need the DbConfigName any more... can be gone from the API in the other files too.